tools/utils.py

```diff
@@ -44,11 +44,11 @@ def wait_for_health_check(instance_ip, timeout_seconds=None):
 
 def check_meilisearch_version(instance_ip, version):
     resp = requests.get(
-        "http://{}/version".format(instance_ip)).json()
+        f"http://{instance_ip}/version").json()
```

Comment:
Here you are using double quotes `"` but above 

>             resp = requests.get(f'http://{instance_ip}/health')

You are using simple quotes. Is this what is expected?

Reply:
Good catch I forgot this one